Automation/src/multiple-ec2-instances/ec2.py

```diff
@@ -0,0 +1,28 @@
+import boto3,os
+from dotenv import load_dotenv
+load_dotenv()
+
+#Load env variables
```

Comment:
Please add a comment describing what the script is about and how to use it 😄

Reply:
hey I have added the steps in the ec2.py file , if I should add in any other way please let me know